### PR TITLE
JS/TS client sample

### PR DESCRIPTION
**Pull request by worksofliam**
* [x] Initial sample app finished
* [x] Sample app tested
* [x] Stop using local copy of mapepire-js with npm version
   * Waiting for new version to be published

### CODE REVIEW

File: typescript/src/ile/sum.sql [hunk 1]
```diff
@@ -0,0 +1,5 @@
+create or replace procedure X.sumpgm (IN numa INT, IN numb INT, OUT result INT) 
+LANGUAGE RPGLE  
+EXTERNAL NAME X.SUM GENERAL;
+
+call X.SUMPGM(1, 2, default);
```
Comment (SanjulaGanepola) [inline typescript/src/ile/sum.sql:5]: Updated `X` to `TESTING` and ran this using the `RUNSQLSTM` action, but I am getting this:
```
SQL7989   0       1  Position 1 Procedure SUMPGM was created in TESTING.
SQL0469  30       5  Position 1 IN, OUT, or INOUT not valid for parameter 3
                      in procedure SUMPGM in TESTING.
```

Am I doing something obviously wrong?

Comment (worksofliam) [reply typescript/src/ile/sum.sql:5]: Pass in `0` instead of `default`. It might work then.

Comment (SanjulaGanepola) [reply typescript/src/ile/sum.sql:5]: Tried that as well. Gives the same error

Comment (worksofliam) [reply typescript/src/ile/sum.sql:5]: Yeah same here. Must only be callable with a binding parameter (see `routes/root.ts`)

Comment (worksofliam) [reply typescript/src/ile/sum.sql:5]: @SanjulaGanepola This worked:

```sql
create or replace variable liama.outparm int default 0;
call LIAMA.SUMPGM(1, 2, liama.outparm);
```

**Comment by jonnyz32** [comment]
Looks good, can we add a setup script/instructions so that we can get liama.sumpgm and liama.people setup on our ibm i more easily?

File: typescript/src/ile/sum.sql (hunk 1 above)
Comment (SanjulaGanepola) [reply typescript/src/ile/sum.sql:5]: Yup that fixed it, thanks

File: javascript/README.md [hunk 2]
```diff
@@ -1,7 +0,0 @@
-# Javascript Sample Projects
-
-These sample projects leverage the [mapepire-js](https://github.com/Mapepire-IBMi/mapepire-js) client SDK:
-
-* `<project-1>`: `<project-1-description>`
```
Comment (SanjulaGanepola) [inline javascript/README.md:5]: Should we move this project into a subdirectory under the typescript folder so other projects can be added later and then add this file back?

**Review by SanjulaGanepola — APPROVED** [review]
Looks good, sample works as expected

File: javascript/README.md (hunk 2 above)
Comment (worksofliam) [reply javascript/README.md:5]: Yes. But, let's do that in a new PR.

**Pull request merged**